infra/stack/tfvars/values.tfvars

```diff
@@ -0,0 +1,8 @@
+label_name          = "nomad"
+region              = "ap-south-1"
+kubernetes_version  = "1.21"
+desired_size        = 1
+instance_type       = "t2.small"
```

Comment:
FYI `t3.*` instance family has better hardware and is cheaper than `t2.*`

Reply:
Yea, make sense will update the instance-type to t3